frontend/public/components/utils/breadcrumbs.ts

```diff
@@ -14,18 +21,34 @@ export const getBreadcrumbPath = (match: any, customPlural?: string) => {
   return `/k8s/cluster/${customPlural || match.params.plural}`;
 };
 
-export const breadcrumbsForDetailsPage = (kindObj: K8sKind, match: any) => () =>
-  kindObj.apiGroup === 'config.openshift.io' && match.params.name === 'cluster'
-    ? breadcrumbsForGlobalConfig(i18next.t(kindObj.labelKey) || kindObj.label, match.url)
-    : [
-        {
-          name: i18next.t(kindObj.labelPluralKey) || kindObj.labelPlural,
-          path: getBreadcrumbPath(match),
-        },
-        {
-          name: i18next.t('details-page~{{kind}} details', {
-            kind: i18next.t(kindObj.labelKey) || kindObj.label,
-          }),
-          path: `${match.url}`,
-        },
-      ];
+export const breadcrumbsForDetailsPage = (kindObj: K8sKind, match: any) => () => {
+  if (kindObj.apiGroup === 'config.openshift.io' && match.params.name === 'cluster') {
+    return breadcrumbsForGlobalConfig(i18next.t(kindObj.labelKey) || kindObj.label, match.url);
```

Comment:
I wouldn't have the logic here. Just use `breadcrumbsFor` in the role and role bindings pages.

Reply:
Ok, I can use `breadcrumbsFor` for the Role and ClusterRole details pages which use [/frontend/public/components/RBAC/role.jsx](https://github.com/openshift/console/blob/997a61229e6a910210f7861ed23a85d339f2735b/frontend/public/components/RBAC/role.jsx#L244).

However, RoleBindings and ClusterRoleBindings use the [DefaultDetailsPage](https://github.com/openshift/console/blob/f8ffd52597c86737547b2935063f71acd0ca59c7/frontend/public/components/default-resource.jsx#L143).  I can move the 'Bindings' breadcrumb logic to DefaultDetailsPage pass in a `breadcrumbsFor`, or create a new details page for RoleBindings and ClusterRoleBindings, or leave `Binding breadcrumb` logic in `breadcrumbsForDetailsPage()`.

Reply:
Hi @spadgett, added `breadcrumbsFor` in `/frontend/public/components/RBAC/role.jsx` for the ClusterRole details pages.
Updated logic in `breadcrumbs.ts` for ClusterRoleBinding breadcrums.

Reply:
I think it would be better to create a custom details page for ClusterRoleBindings, even if only to set the breadcrumbs.

Reply:
Added custom details page for ClusterRoleBindings